### PR TITLE
vapor: init at 0.2.3

### PR DESCRIPTION
**Pull request by leenaars**
Vapor is a LÖVE distribution client, allowing easy download of LÖVE 0.8 games.

### CODE REVIEW

File: pkgs/games/vapor/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url =
+    "https://github.com/josefnpat/${pname}/releases/download/${version}/${pname}_${commitid}.love";
+    sha256 = "0w2qkrrkzfy4h4jld18apypmbi8a8r89y2l11axlv808i2rg68fk";
+  };
+
+  nativeBuildInputs = [ lua love ];
+  buildInputs = [ makeWrapper ];
```
Comment (hrdinka) [inline pkgs/games/vapor/default.nix]: These are switched. `nativeBuildInputs` should be `buildInputs` and the other way around.

**Comment by leenaars** [comment]
@hrdinka:  I've swapped them ...

**Comment by hrdinka** [comment]
Great, I have just built and tested it and everything works as expected :+1:
The derivation lacks a maintainer. Since you are already maintaining your recent love related packages I guess that you have just missed adding yourself. Further `unzip` isn't used anywhere and can be removed from the argument list in line 1.

**Comment by leenaars** [comment]
@hrdinka: added maintainer and removed unzip.

**Comment by hrdinka** [comment]
Built & tested locally (linux x86_64). Thanks for contributing! :+1:

**Pull request merged**